cli/run/batchWarnings.ts

```diff
@@ -270,6 +266,20 @@ const deferredHandlers: {
 			});
 		},
 		priority: 1
+	},
+
+	EMPTY_BUNDLE: {
+		fn: warnings => {
+			title(
+				`Generated${warnings.length === 1 ? ' an' : ''} empty ${
+					warnings.length > 1 ? 'chunks' : 'chunk'
+				}`
+			);
+			warnings.forEach(warning => {
+				stderr(warning.chunkName as string);
+			});
+		},
+		priority: 1
```

Comment:
It would be good to also add a CLI test for this
1. This will fix the coverage problem, as this code is currently not covered, and
2. It would be nice to see what this looks like when there are several chunks. Also, instead of calling `stderr` multiple times, why not pre-format it and call it once, which would give more formatting options, e.g.

```js
stderr(warnings.map(warning => warning.chunkName).join(', '))
```